bazel/external/libprotobuf_mutator.BUILD

```diff
@@ -16,3 +16,20 @@ cc_library(
     visibility = ["//visibility:public"],
     deps = ["//external:protobuf"],
 )
+
+cc_library(
+    name = "libprotobuf_mutate",
```

Comment:
How come we need this now? Weren't we depending upon LPM before already?

Reply:
Just the macro, not the mutator. Let me see if I can clean that up, because I don't think that's the best naming scheme.

Reply:
Excluding this LPM part seems reasonable.

Reply:
Done!